client/components/progress-bar/index.jsx

```diff
@@ -59,19 +54,23 @@ export default class ProgressBar extends PureComponent {
 	}
 
 	renderBar() {
-		const title = this.props.title ? (
-			<ScreenReaderText>{ this.props.title }</ScreenReaderText>
-		) : null;
+		const { color, title, total, value } = this.props;
 
 		const styles = { width: this.getCompletionPercentage() + '%' };
-		if ( this.props.color ) {
-			styles.backgroundColor = this.props.color;
+		if ( color ) {
+			styles.backgroundColor = color;
 		}
 
 		return (
-			<div className="progress-bar__progress" style={ styles }>
-				{ title }
-			</div>
+			<div
+				aria-valuemax={ total }
+				aria-valuemin={ 0 }
+				aria-valuenow={ value }
+				aria-valuetext={ title || '' }
```

Comment:
I'm not sure `aria-valuetext` is the right choice for the `title` prop :thinking: Both the fallback value (`Your setup list`) and the JP checklist one (`Your Jetpack setup progress`) are "static" -- they don't reflect the current value of the progress bar.

See also https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_progressbar_role, which says

> Note: Assistive technologies generally will render the value of aria-valuenow as a percent of the range between the value of aria-valuemin and aria-valuemax, unless aria-valuetext is specified. It is best to set the values for aria-valuemin, aria-valuemax, and aria-valuenow in a manner that is appropriate for this calculation.

Sounds like `aria-valuetext` would be better used for something like `3/6` (reflecting what step we're currently at) in our case.

Not sure if there's a better `aria` attr for `title` :grimacing:

Reply:
(To highlight an important aspect, setting `aria-valuetext` to something static might mean that screen readers don't inform users about the current value of the progress bar at all.)

Reply:
BTW there's an [HTML5 `<progress />`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress) element :grimacing: So maybe we should go all semantic?

Reply:
Thanks, all very good and valid points!

It might be more straightforward (considering the component is used all over the place in various ways) to leave out `aria-valuetext` for now. 🤔

Reply:
Dropped in https://github.com/Automattic/wp-calypso/pull/33016/commits/dabe49be666fb6bab57e50971f612e15f6283a6c